core_impl/src/main/java/io/opencensus/implcore/stats/MutableAggregation.java

```diff
@@ -223,10 +223,9 @@ long getCount() {
   /** Calculate distribution stats on aggregated {@code MeasureValue}s. */
   static final class MutableDistribution extends MutableAggregation {
 
-    private double sum = 0.0;
+    private double mean = 0.0;
```

Comment:
What is the reason for this change?  I thought that the math was simpler when we had a sum field.

Reply:
Because to calculate sum of squared deviations, we still need to calculate mean first, and also two deltas from mean: https://github.com/census-instrumentation/opencensus-java/blob/master/core_impl/src/main/java/io/opencensus/implcore/stats/MutableAggregation.java#L394

In this case I think it's easier to store mean directly.

Reply:
I only meant the change to remove the sum.  I thought it was simpler to store both the sum and the mean.

Reply:
Make sense, I've put back this field.